kubernetes/test-deployment.yml

```diff
@@ -0,0 +1,67 @@
+# TODO:
```

Comment:
Please, you may justify why this is showing TODO, is the coding part over yet?

Reply:
Actually, this file was just same resource manifests for testing out our deployment. Should I delete it?

Reply:
It will be superceded by the helm chart in PR #12

Reply:
Ok, If this file is unnecessary, please just delete it.

Reply:
Okay will do

Reply:
Deleted